src/Libraries/Inscriptions/Envelope.ts

```diff
@@ -60,16 +61,26 @@ export class Envelope {
    * @param tx - Raw bitcoin transaction to search for an inscription envelope
    */
   static fromTransaction(tx: RawTransaction) {
-    const [data, oip] = getEnvelopeDataFromTx(tx) ?? [];
-    if (data) {
-      return new Envelope(tx.txid, data, oip);
+    const envelopes = getEnvelopesDataFromTx(tx);
+    if (envelopes) {
```

Comment:
should we check for `.length > 0` instead? because `[]` will pass the if check

Reply:
okii